README.md

```diff
@@ -116,7 +116,7 @@ Specifically, it loads configuration from:
 
  * `.conjurrc` files, located in the home and current directories, or at the
     path specified by the `CONJURRC` environment variable.
- * Reads `/etc/conjur.conf` as a `.conjurrc` file.
+ * Reads `/etc/conjur.conf` as a `.conjurrc` file. If the platform is Windows, the path is `C:\Windows\conjur.conf` instead.
```

Comment:
```suggestion
 * Reads the `.conjurrc` file from `C:\Windows\conjur.conf` on Windows and from `/etc/conjur.conf` otherwise.
```
What about wording it something like this instead?

Reply:
@izgeri I changed the wording but I wanted to retain the most common platforms first in the description so take a look and let me know if it's good